apps/src/templates/teacherHomepage/ManageSections.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import CollapsibleSection from './CollapsibleSection';
+// import SectionsTable from './SectionsTable';
```

Comment:
nit: don't commit commented-out code unless there's a comment explaining why, because it won't be clear to later viewers why it's commented-out.  But better to just not commit commented-out code at all.